Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
+        "description": "Your first steps to set up powerful C++ tools in a lightweight, performant editor!",
+        "steps": [
+          {
+            "id": "verify.compiler.mac",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. [Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isMac",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing Clang on macOS",
+              "path": "walkthrough/install-clang-macos.md"
+            }
+          },
+          {
+            "id": "verify.compiler.linux",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isLinux",
+            "doneOn": {
+              "command": "C_Cpp.CheckForCompiler"
+            },
+            "media": {
+              "altText": "markdown file with instructions for installing GCC on Linux",
+              "path": "walkthrough/install-gcc-linux.md"
+            }
+          },
+          {
+            "id": "verify.compiler.windows",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
```

Comment:
In the remote scenario, the remote computer's compilers are used...not sure if that's worth mentioning or if it would just complicate things.

Reply:
Yeah, let's use "system" instead of "computer" because that makes it more versatile.

Reply:
Yeah, I noticed other text below used "system" too, so that'd be more consistent.